### PR TITLE
Add a popup modal upon clicking download button

### PR DESCRIPTION
**Pull request by kgcthu23**
Hola amigo

### CODE REVIEW

File: components/DownloadPopup.jsx [hunk 1]
```diff
@@ -11,9 +11,20 @@ export default function DownloadPopup({ isOpen }) {
   return (
     <>
       {/* <div className="download-overlay w-full h-full fixed bg-gray-800 top-0 left-0 opacity-50 z-100"></div> */}
-      <div className="download fixed w-80 bg-gray-100 p-10 h-40 top-1/2 left-1/2 transform -translate-x-1/2 -translate-y-1/2">
-        <div className="download__container"></div>
+      <div className='download fixed w-80 bg-gray-100 p-10 h-40 top-1/2 left-1/2 transform -translate-x-1/2 -translate-y-1/2'>
+        <div className='download__container mx-3'>
+          <div className='flex flex-row translate-y-1/3 justify-center align-center space-x-3'>
+            {result.map((dInfo, index) => (
+              <button
```
Comment (KaungKZ) [inline components/DownloadPopup.jsx]: Add width and height to `a href`, not the button

File: components/DownloadPopup.jsx [hunk 2]
```diff
@@ -11,9 +11,20 @@ export default function DownloadPopup({ isOpen }) {
...
+                key={index}
+                className='py-3 px-6 bg-green-400 rounded text-white hover:bg-green-500 transition flex items-center text-base'
+              >
+                <a href={dInfo.url}>{dInfo.quality}</a>
```
Comment (KaungKZ) [inline components/DownloadPopup.jsx]:
```suggestion
         <a href={dInfo.url} target="_blank">{dInfo.quality}</a>
```

**Review by KaungKZ — COMMENTED** [review]
Try follow modal design of [this](https://yts.mx/movies/the-fallout-2021)

File: components/DownloadPopup.jsx (hunk 1 above)
Comment (CryoMilo) [reply components/DownloadPopup.jsx:18]: oh noh. why scold my boi kst

**Pull request merged**